### PR TITLE
fix: issue with md files resolving images

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: docs/images/download/npm-ed1c24.svg
Comment (mfranzke) [inline docs/images/download/npm-ed1c24.svg:1]: That's not the same image as it doesn't include the version information as in e.g. https://img.shields.io/npm/v/@db-ui/ngx-components.svg?color=rgb%28237%2C%2028%2C%2036%29

Comment (nmerget) [reply docs/images/download/npm-ed1c24.svg:1]: Yes I removed the dynamic information. I think it's not necessary

**Comment by mfranzke** [comment]
@nmerget I'm struggling whether it makes sense to check in the transformed files, but instead keep in the URLs, and only have those rewrites being done during build for the GitHub Pages exclusively – both because we only need them on this environment and we could even also keep dynamic contents like we would need them for items like this: https://github.com/db-ui/mono/pull/1283#discussion_r1266416265

**Comment by nmerget** [comment]
> @nmerget I'm struggling whether it makes sense to check in the transformed files, or instead keep in the URLs, and only have those rewrites being done during build for the GitHub Pages exclusively – both because we only need them on this environment and we could even also keep dynamic contents like we would need them for items like this: [#1283 (comment)](https://github.com/db-ui/mono/pull/1283#discussion_r1266416265)

So it's fine to have those links (shield.io) on the default readme on GitHub?

**Review by mfranzke — DISMISSED** [review]

**Review by mfranzke — APPROVED** [review]

**Pull request merged**